modules/change-stream/change-stream.js

```diff
@@ -5,6 +5,7 @@
 
 angular.module('ls.ChangeStream',[]).factory('createChangeStream', ['$rootScope', function($rootScope) {
   function createChangeStream(eventSource, $scope) {
+    var stream = require('stream')
```

Comment:
our test phantomjs browser doenst have requirejs or such package managers, so it cant do `require('stream')`